engine/src/font/atlas/bridge.cpp

```diff
@@ -0,0 +1,52 @@
+#include <msdf-atlas-gen/msdf-atlas-gen.h>
+
+#include <cubos/engine/font/atlas/atlas.hpp>
+#include <cubos/engine/font/atlas/bridge.hpp>
+
+bool cubos::engine::FontAtlasBridge::load(Assets& assets, const AnyAsset& handle)
+{
+    double minimumScale = 24.0;
+    double pixelRange = 4.0;
+    double miterLimit = 1.0;
+    AnyAsset fontAsset{};
+    {
+        auto meta = assets.readMeta(handle);
+        std::optional<std::string> optMinimumScale = meta->get("minimumScale");
+        if (optMinimumScale.has_value())
+        {
+            minimumScale = std::stod(optMinimumScale.value());
+        }
```

Comment:
```suggestion
        if (auto optMinimumScale = meta->get("minimumScale"))
        {
            minimumScale = std::stod(*optMinimumScale);
        }
```
Cooler syntax :relieved: